### PR TITLE
[Merged by Bors] - add two to_additive name translations

### PR DESCRIPTION
**Pull request by fpvandoorn**
* Remove manual translations that are now guessed correctly
* Fix some names that were incorrectly guessed by humans (and in one case fix the multiplicative name). Add deprecations for all name changes.
* Remove a couple manually additivized lemmas.

---

Is this the right way to deprecate?

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by fpvandoorn** [comment]
This will conflict with #10831. If one is merged, the other PR should merge master.

**Comment by adomani** [comment]
I can't look carefully at the translations, but the deprecations look correct to me.  The only difference with what I've seen is that the date appeared at the end, but I do not think that there is a style requirement for that.

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 1]
```diff
@@ -278,7 +281,7 @@ section Classical
 
 open Classical
 
-@[to_additive IsAddCyclic.card_pow_eq_one_le]
+@[to_additive IsAddCyclic.card_nsmul_eq_zero_le]
```
Comment (adomani) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:284]: Good catch: this is a very subtle name translation!

(Same goes for the one below!)

**Review by adomani — APPROVED** [review]
Thanks!

I checked the diff and the substitution on `to_additive`: everything looks good to me, although I am not sure that I would have caught the subtle word-splitting requiring that `..._one_le` should not have been changed to `..._nonneg`.

**Comment by fpvandoorn** [comment]
Yes, I only found it myself by investigating why `to_additive` did *not* warn me about the superfluous name.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by fpvandoorn** [comment]
This looks like a spurious error

bors merge

**Pull request closed without merging**